$OpenBSD: patch-configure_in,v 1.2 2009/04/14 20:33:11 kili Exp $

XXX disabling PIE is a workaround to let dbus work on arm.

--- configure.in.orig	Wed Jan  7 01:30:26 2009
+++ configure.in	Tue Apr 14 12:46:30 2009
@@ -253,19 +253,6 @@ if test "x$GCC" = "xyes"; then
      ;;
   esac
 
-  case " $CFLAGS " in
-  *[\ \	]-fPIE[\ \	]*) ;;
-  *) if cc_supports_flag -fPIE; then
-        PIE_CFLAGS="-fPIE"
-        if ld_supports_flag -z,relro; then
-           PIE_LDFLAGS="-pie -Wl,-z,relro"
-        else
-           PIE_LDFLAGS="-pie"
-        fi
-     fi
-     ;;
-  esac
-  
   ### Disabled warnings, and compiler flag overrides
   
   # Let's just ignore unused for now
@@ -282,10 +269,6 @@ if test "x$GCC" = "xyes"; then
   *[\ \	]-Wno-sign-compare[\ \	]*) ;;
   *) CFLAGS="$CFLAGS -Wno-sign-compare" ;;
   esac
-  case " $CFLAGS " in
-  *[\ \	]-Wno-pointer-sign[\ \	]*) ;;
-  *) CFLAGS="$CFLAGS -Wno-pointer-sign" ;;
-  esac  
   
   # http://bugs.freedesktop.org/show_bug.cgi?id=19195
   case " $CFLAGS " in
